### DISCUSSION TITLE
Where is defined in the GUI k and e for a k-Epsilon turbulent incompressible model?

### DISCUSSION THREAD

**Discussion by SergioP1975** [nextfoam/baram#132] [Q&A]
I´m triyng to made a simple model using the turbulent incompresible Epsilon k model, but I´m struggling to input my k/e values, no matter what I enter in the GUI (there are several places where I guess that could be entered), the dictionaries are written with other values. I even handwrite the k/epsilon files before running the solver...and again are changed!

Regards, and thanks in advance.

**Comment by kdg5424** [comment]
Hello @SergioP1975. This is Donggyu from NEXTFOAM.

You can input k/e values in BCs like the below picture.
![image](https://github.com/user-attachments/assets/3aec99d7-920a-4826-8416-899bdf1f2b71)

Do you mean you can't change with this option?

Best regards

**Reply by SergioP1975** [reply]
I swear that I have changed that, tested and the values in the dictionaries were the same as if I have used the option "intensity and viscosity ratio", no matter what I enter it takes the same value. I have tried even changing it in the Cell Zone Conditions/Region/Source and Fixed values, but again the writted value on the dictionaries were not that ones.

 I will test again in different computers.

**Reply by kdg5424** [reply]
Well it's really weird. Could you send your case?
Please attach here or if it's complicated, please let me know.

Best regards

Donggyu Kim

**Reply by SergioP1975** [reply]
I have tested today again, and the values that are writted in the k/epsilon dictionaries are not the same as the one entered in the GUI.
I´m tring to replicate the case in this Youtube video:

https://youtu.be/DLsjq05ZhJk?t=546

I have made a first case "by hand" taking some basic example and then adding the k/epsilon/nut files, and the results are the same as in the video (but using a full model), is the zip file begining with 01.

After that I use Boram to create a second model, but now using symmetry bc, but I can´t find a way to reproduce the results of the first case, and the only difference that I see is in the values of k/e. This is the zip file 02

This is a link to download the two cases

https://drive.google.com/drive/folders/19oC2lKjhE-gRTn3zftzkM_YzsVI3VcFU?usp=sharing

Best regards, and thanks in advance.

**Reply by kdg5424** [reply]
Hi. Can you check your units? In the youtube video that you mentioned, it looks like the units of the length is '**mm**'.
But the units in BARAM looks like '**meter**'.
![image](https://github.com/user-attachments/assets/f40d30c9-569a-469e-8e6c-b77ec09e2c50)

And there is something problem with initialization.
If you configure initilaization like this, in 0 folder the value of turbulence will be calculated with this value.
![image](https://github.com/user-attachments/assets/ba9fd3c0-746b-46e8-bad8-425ae6ff75c6)

I think the reason why the results are different is because of the units. I'll change and calculate again with right units and initialization. And let you know

Best regards

**Reply by SergioP1975** [reply]
Thanks a lot for the support! I´m begining with CFD (being in FEA for years) and found Baram a very valuable tool.

**Comment by kdg5424** [comment]
Hello @SergioP1975. This is Donggyu.
I think I get the right result compared to ANSYS Fluent.

This image is the result with BARAM.
![image](https://github.com/user-attachments/assets/9722dc03-935c-4ce5-86c9-000a4b311654)

And this image is the result with ANSYS Fluent.
![image](https://github.com/user-attachments/assets/6e6a902f-b2d9-47fe-a94c-10c0159c1681)

Although I didn't plot the velocity profile, compared to these images I guess the result is right.
I just scaled mesh 0.001 times and used '**Second Order Upwind**'. You can scale it in Mesh - Scale.
And I didn't change the value of initialization. Just used default value.

![image](https://github.com/user-attachments/assets/a672f754-73e6-4273-b509-9c365c5c1e2f)
![image](https://github.com/user-attachments/assets/4b929941-18ac-4efb-8c4c-642880f85fe6)
![image](https://github.com/user-attachments/assets/4a62a627-0a5d-467b-96c0-b96ff9145ce4)

I think you can follow it very easily and if you have any other problems, please let me know.

Best regards

Donggyu Kim

**Reply by kdg5424** [reply]
And one more this is the residuals of the simulation.
![image](https://github.com/user-attachments/assets/9ec14f10-d9df-4595-9904-47ebfcf8c5fb)

**Reply by SergioP1975** [reply]
Thanks a lot for your help @kdg5424, now I was able to reproduce exactly the reference model.

Best regards.

**Reply by kdg5424** [reply]
I'm glad to hear that.

Best regards
